### PR TITLE
DOC-4543

### PR DESCRIPTION
**Pull request by jamesnocentini**
https://issues.couchbase.com/browse/DOC-4543

### CODE REVIEW

**Comment by jamesnocentini** [comment]
Reviewers: @adamcfraser

File: modules/ROOT/pages/release-notes.adoc [hunk 1]
```diff
@@ -24,6 +24,13 @@ The recommended approach to perform operations on a bucket dedicated to Couchbas
 - {url-issues-sync}/3740[*#3740*] Persist temporary revision body backups on import
 - {url-issues-sync}/3756[*#3756*] DCP Feed does not seed from all nodes
 
+*Known Issues*
+
+- {url-issues-sync}[*N/A*] Index Panic issue: the indexer throws an `index out of range` runtime error.
+* *Workaround:* disable xref:2.1@index.adoc#guaranteed-availability[GSI indexing].
```
Comment (adamcfraser) [inline modules/ROOT/pages/release-notes.adoc:31]: Might be better to phrase this as 'Query failures when querying xattrs with non-covering index', as there are multiple underlying query service issues fixed in 5.5.3:

https://issues.couchbase.com/browse/MB-31990
https://issues.couchbase.com/browse/MB-32120

File: modules/ROOT/pages/release-notes.adoc [hunk 2]
```diff
@@ -24,6 +24,13 @@ The recommended approach to perform operations on a bucket dedicated to Couchbas
...
+- {url-issues-sync}[*N/A*] Sync Gateway channel queries to the Couchbase Server may time out.
+* *Workaround:* disable xref:2.1@index.adoc#guaranteed-availability[GSI indexing].
```
Comment (adamcfraser) [inline modules/ROOT/pages/release-notes.adoc:33]: https://github.com/couchbase/sync_gateway/issues/3822

**Review by adamcfraser — CHANGES_REQUESTED** [review]
Added ticket links, and a comment on better describing the server issue.

**Pull request merged**